### DISCUSSION TITLE
Commands not getting deleted from chats

### DISCUSSION THREAD

**Discussion by rayanfer32** [phaticusthiccy/WhatsAsenaDuplicated#124] [Bug Discussion] [answered]
**Describe the bug**
Commands  are not getting deleted automatically from chats, it spoils the experience of bot. 

##

**Expected behavior**
Commands should be deleted from the chat after they are executed.
##

**Users, please complete the following actions.**
- WhatApp Package Name - 2.21.13.28
- WhatsAsena Version (latest)
- Work Type (Private)

**Comment by phaticusthiccy** [comment, marked as answer]
Hi, I am really working on it. After x.x.11.x version, our message status mark as "ERROR" 
We need "SERVER_ACK" to complete function.  I can't update baileys version because it will bring sooo many issues. It took 3 Months to fix it and ı still don't have any solution. When ı find a way to fix it, ı will update quickly. 
Until then, I'll try to find a solution.

**Reply by rayanfer32** [reply]
Oh I understand, nice work there. It took me a whole day to explore the features of this amazing project.

I found another fork https://github.com/test-w5/WhatsAsenaDuplicated where this issue is fixed ... but it was very annoying to realize it was sending messages to other contacts and groups .. it was a bug highlight in issues
